utils/preferences/src/extensions/unicode/keywords/calendar.rs

```diff
@@ -0,0 +1,34 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+#![allow(non_snake_case)]
+
+use crate::enum_keyword;
+
+// https://github.com/unicode-org/cldr/blob/main/common/bcp47/calendar.xml
+enum_keyword!(IslamicCalendar {
+    "umalqura" => Umalqura,
+    "tbla" => Tbla,
+    "civil" => Civil,
+    "rgsa" => Rgsa
+});
+
+enum_keyword!(Calendar {
```

Comment:
Observation: this list is missing some aliases. Some of the others might be, too, but I haven't checked. We should do a thorough review of all these enums before 2.0